lib/cereal/serializer.ex

```diff
@@ -171,9 +175,25 @@ defmodule Cereal.Serializer do
     end
   end
 
-  defp normalize_relationship_opts(opts, _) do
+  defp normalize_relationship_opts(opts, _env) do
     quote do
       unquote(opts) |> Enum.into(%{})
     end
   end
+
+  def aliases(opts) do
```

Comment:
This one's mysterious. Can we either add a @doc or make it private if it could be?

Reply:
Damn, this function wasn't meant to be in the PR. It's a leftover from when I thought it'd be a good idea to look for all aliases in the association's options, rather than the lower-touch approach of just the `:serializer` option.

Sorry about that.